_layouts/software-details.html

```diff
@@ -388,8 +388,15 @@ <h2>
                 <p>
                   <span class="label">{{ t.software.contract_with }}</span>
                   {% for contractor in page.publiccode.maintenance.contractors %}
-                  <a href="{{ contractor.website }}" class="until">{{ contractor.name }}</a> {{ t.software.until }} {{
-                  contractor.until}}
+                    {% if contractor.website != nil and contractor.website != "" %}
+                    <a href="{{ contractor.website }}" class="until">
+                      <span class="icon it-external-link d-inline-block"></span>
```

Comment:
As stated before, I wouldn't introduce this change in this PR since it's not consistent with the rest of the catalogue. If we decide to go the hard way and switch all external links to use such icon we might do it consistently in a separate issue + PR.

Reply:
Should I remove this one too? 
On the issue was reported `Furthermore, it could be nice to follow what the mockup suggests: since this is an external link, it's better to notify the user with some sort of icon placed next to the link`

Reply:
Yes, let's remove this and open a new PR to solve the other issue (https://github.com/italia/developers.italia.it/issues/431)